script/libtest.bash

```diff
@@ -322,4 +324,14 @@ test_topic_deletion() {
      exit 200
     fi
 }
+sts_restart() {
+    local num=1
+    local topic=$RANDOM
+    kubeless topic create $topic
```

Comment:
I would move the `topic create` and `topic list` outside the `restart` function, it is not related to the restart process

Reply:
Makes sense.